gips/inventory.py

```diff
@@ -250,12 +252,16 @@ def __init__(self, dataclass, spatial, temporal, products=None, fetch=False, **k
         self.temporal = temporal
         self.products = dataclass.RequestedProducts(products)
 
+        self.update = update
+
+        print "datainventory"
```

Comment:
Yes, there are also “from pdb import set_trace” statements, sorry.

> On Jul 5, 2016, at 10:19 AM, ags-tolson <notifications@github.com> wrote:
> 
> In gips/inventory.py <https://github.com/Applied-GeoSolutions/gips/pull/53#discussion_r69570661>:
> 
> > @@ -250,12 +252,16 @@ def __init__(self, dataclass, spatial, temporal, products=None, fetch=False, **k
> >          self.temporal = temporal
> >          self.products = dataclass.RequestedProducts(products)
> >  
> > +        self.update = update
> > +
> > +        print "datainventory"
> Is this supposed to stay in? Looks like a stray debugging statement.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/Applied-GeoSolutions/gips/pull/53/files/3a9deaff37b928cc1fba52d1cdb2083f4a8127db#r69570661>, or mute the thread <https://github.com/notifications/unsubscribe/ADD4XOap_JflyYbBAIymK-s56b6zSVe9ks5qSmfagaJpZM4JDj03>.
>